pkg/handlers/docker/mutation/customimage/inject.go

```diff
@@ -149,6 +159,10 @@ func (h *customImagePatchHandler) Mutate(
 		return err
 	}
 
+	if h.variableName == workerconfig.MetaVariableName {
+		return nil
+	}
+
```

Comment:
Not a fan of this

Reply:
This feels like they should be separate handlers, one for control plane and one for workers. WDYT?

Reply:
👍 yep split into multiple handlers